README.md

```diff
@@ -60,6 +60,49 @@ Some highlights:
 Take a tour by our [documentation](http://port.us.org/features.html) site to
 read more about this.
 
+## Deploying
+
+### Containerized
+
+In the `examples` directory you can find different containerized deployment
+examples. All these examples are using the `opensuse/portus` image, which can be
+found on Docker Hub, and they are defined in
+[opensuse/docker-containers](https://github.com/openSUSE/docker-containers/tree/master/derived_images/portus). We
+have the following tags:
+
+- `head`: git's master branch packaged and curated as if it was
+  production-ready. This is convenient for people that want to be on the
+  bleeding edge and they want to test the latest features.
```

Comment:
`Portus' master branch [...] This is convenient for people who want to be on the bleeding edge and want to test the latest features.`